VoteCounter/src/GUI.cs

```diff
@@ -27,10 +30,55 @@ private void btnGenerateVoteCount_Click(object sender, EventArgs e)
 
             if (Int32.TryParse(txtStartingPost.Text, out startingPostNumber) && Int32.TryParse(txtEndingPost.Text, out endingPostNumber))
             {
+                //make a List
```

Comment:
Maybe? Is there a way to do it without just doing "List<Player> players = createPlayerList();" ?
Because there's not much tidy difference in it.